vms/evm/sync/customrawdb/errors.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package customrawdb
+
+import "errors"
+
+var (
+	// ErrEntryNotFound indicates the requested key/value was not present in the DB.
+	ErrEntryNotFound = errors.New("customrawdb: entry not found")
```

Comment:
👍 I've seen this package name prefixing in stdlib and although we don't do it in our codestyle I think it seems like a nice suggestion since it adds meaningful context to wrapped errors